src/guiFormSpecMenu.cpp

```diff
@@ -1604,14 +1593,14 @@ void GUIFormSpecMenu::parseTooltip(parserData* data, std::string element)
 	std::vector<std::string> parts = split(element,';');
```

Comment:
Probably, however I did not touch this code so I will not change it in that PR; I could however do another one for that.